cypress/support/grist_mocks.js

```diff
@@ -27,24 +27,11 @@ Cypress.Commands.add('mockGristRecords', (resourceId, elements = []) => {
   ).as(`get_grist_${resourceId}_records`)
 })
 
-Cypress.Commands.add('mockGristRecord', (resourceId, element) => {
-  cy.intercept(
-    'GET',
-    `https://grist.numerique.gouv.fr/api/docs/*/tables/${resourceId}/records?filter=${JSON.stringify({ id: [element.id] })}`,
-    {
-      statusCode: 200,
-      body: {
-        records: [element]
-      }
-    }
-  ).as(`get_grist_${resourceId}_record_${element.id}`)
-})
-
 Cypress.Commands.add('mockGristRecordsByIds', (resourceId, elements) => {
   const ids = elements.map((el) => el.id)
   cy.intercept(
     'GET',
-    `https://grist.numerique.gouv.fr/api/docs/*/tables/${resourceId}/records?filter=${JSON.stringify({ id: ids })}`,
+    `https://grist.numerique.gouv.fr/api/docs/*/tables/${resourceId}/records?filter=%7B%22id%22:%5B${ids.join(',')}%5D%7D`,
```

Comment:
And this was not properly encoded, so I ended up putting the encoded character directly, even if it's not ideal to read.